frontend/src/index.js

```diff
@@ -1,9 +1,31 @@
+/* eslint-disable prettier/prettier */
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { App } from './components';
+import './index.css';
+import App from './App';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { userReducers } from './reducers';
+import { articleReducers } from './store/reducers';
+
 // import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const store = createStore(
+	userReducers,
+	articleReducers,
```

Comment:
You should only be importing one reducer file with this. It picks up the second call as a prop for something else. See comment in src/index.js line 11 on how to handle this cleanly.